examples/llamaindex/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>Is it worth adding a link or two here that can point users off to relevant pre-reading? e.g. make <code>LlamaIndex</code> link to their site, and make <code>RAG</code> link to something with background  info on RAG?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>